gatsby_fp/src/components/fieldComponents/intellCont.js

```diff
@@ -0,0 +1,195 @@
+import React from "react"
+
+export default function IntellCont ({intellContList}){
+  
+  return(   
+    <>
+    <div className="sectiontitle">Selected Publications</div>
+        <div>
+          <ol>
+            {intellContList.map((element, index) => (
+              // <li key={index} dangerouslySetInnerHTML = {{__html: 
+              //   `${element.contype} ${element.contypeother}
+              //   <br>(${element.dtm_pub} ${element.dtd_pub} ${element.dty_pub}). 
+              //   ${element.title}, ${element.title_secondary}, ${element.publisher}
+              //   <br>Issue: ${element.issue}
+              //   <br>Number of Pages: ${element.number_of_pages}
+              //   <br>Status: ${element.status}<br>Refereed:${element.refereed}
+              //   <br>Editorial Reviewed: ${element.editorial_reviewed}
+              //   <br> ISBNISSN: ${element.isbnissn}
+              //   <br>User Reference Creator: ${element.user_reference_creator}`}}>
+              // </li>
+                <li key={index} dangerouslySetInnerHTML ={{__html:` 
```

Comment:
We should avoid using indices as keys. (https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)